subnet/etcdv2/local_manager.go

```diff
@@ -33,7 +33,8 @@ const (
 )
 
 type LocalManager struct {
-	registry Registry
+	registry       Registry
+	previousSubnet ip.IP4Net
```

Comment:
I wanted to avoid changing the `AcquireLease` function to accept additional arguments since that would require the kube manager to also be updated when it does not need a subnet ip (since it is not affected by this).  I could change it but I wanted to avoid changing the interface signatures.